lib/active_record/sql_analyzer/configuration.rb

```diff
@@ -150,15 +150,14 @@ def setup_defaults
         @options[:sql_redactors] = [
           Redactor.new(/\n/, " "),
           Redactor.new(/\s+/, " "),
-          Redactor.new(/IN \([^)]+\)/i, "IN ([REDACTED])"),
-          Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?-?\d+(\.\d+)?/i, " = [REDACTED]"),
+          Redactor.new(/IN \([^)]+\)/i, "IN ('[REDACTED]')"),
+          Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?-?\d+(\.\d+)?/i, " = '[REDACTED]'"),
           Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?x?#{quotedValuePattern}/i, " = '[REDACTED]'"),
-          Redactor.new(/VALUES \(.+\)$/i, "VALUES ([REDACTED])"),
+          Redactor.new(/VALUES \(.+\)$/i, "VALUES ('[REDACTED]')"),
           Redactor.new(/BETWEEN #{quotedValuePattern} AND #{quotedValuePattern}/i, "BETWEEN '[REDACTED]' AND '[REDACTED]'"),
-          Redactor.new(/LIKE #{quotedValuePattern}/i, "LIKE '[REDACTED]'"), 
+          Redactor.new(/LIKE #{quotedValuePattern}/i, "LIKE '[REDACTED]'"),
           Redactor.new(/ LIMIT \d+/i, ""),
           Redactor.new(/ OFFSET \d+/i, ""),
-          Redactor.new(/INSERT INTO (`?\w+`?) \([^)]+\)/i, 'INSERT INTO \1 ([COLUMNS])'),
```

Comment:
Not sure, you could just look at a few INSERT statements to ensure it's
consistent. I suspect it's to ensure that DB migrations don't mess up
everything.

On Wed, Jul 6, 2016 at 3:31 PM, Andrew Ryan Lazarus <
notifications@github.com> wrote:

> In lib/active_record/sql_analyzer/configuration.rb
> <https://github.com/square/active_record-sql_analyzer/pull/5#discussion_r69823564>
> :
>
> >            Redactor.new(/ LIMIT \d+/i, ""),
> >            Redactor.new(/ OFFSET \d+/i, ""),
> > -          Redactor.new(/INSERT INTO (`?\w+`?) \([^)]+\)/i, 'INSERT INTO \1 ([COLUMNS])'),
>
> If the output is a mess without the column list redaction, I'll look at a
> better way to deal with it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/square/active_record-sql_analyzer/pull/5/files/64047c50af4b6718c1d6e3d36c71fe8402926841#r69823564>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AATvvdCQCfGiKxRHUsm-TGitHtHmZBgTks5qTCzZgaJpZM4JFn6r>
> .
>